docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component in order to share files among container components, making it easier to collaborate with other teams during the development process.
```

Comment:
IBM SG: "Use "to" in most cases to avoid wordiness."

Reply:
Hi, @Jake Berger. I was revisiting this PR and realized I missed something important during my peer review. The content talks about adding a component to a devfile, but omits the effect of this action in OpenShift, and then describes the benefit. I'd recommend something more like:
```suggestion
To create a <name the type of volume> in your shared <location of volume>, you can add a volume component to your devfile. This <type of volume> makes it easier for teams to share files and collaborate with each other during the development process.
```
WDYT?

Reply:
A similar approach might also apply to docs/modules/user-guide/partials/proc_adding-a-kubernetes-or-openshift-component-to-a-devfile.adoc

Reply:
Devs, I think Rolfe's suggestion is good, but I honestly wouldn't be able to say what type of component it is. Could any of you give an idea to expand our abstract more?